modules/reactor/src/meshgenerators/FlexiblePatternGenerator.C

```diff
@@ -429,11 +435,28 @@ FlexiblePatternGenerator::FlexiblePatternGenerator(const InputParameters & param
   if (std::count(input_usage_count.begin(), input_usage_count.end(), 0))
     paramError("inputs", "All the input mesh generator names are not used.");
 
+  if (_delete_default_external_boundary_from_inputs)
+  {
+    for (const auto & input_name : _input_names)
+    {
+      auto params = _app.getFactory().getValidParams("BoundaryDeletionGenerator");
+      params.set<MeshGeneratorName>("input") = input_name;
+      params.set<std::vector<BoundaryName>>("boundary_names") = {std::to_string(OUTER_SIDESET_ID)};
```

Comment:
how do we know this boundary is always in the mesh?
since we default to true, it will error if it s not

Reply:
`MooseMeshUtils::getBoundaryID()` does not return `libMesh::BoundaryInfo::invalid_id` if the input is a number (id). In that case, it does not throw any errors. Not sure if this is the intended behavior, it wont error as we use `10,000` here.

Reply:
nice. I m not sure this is intentional either. This works here though, and we can change that later